store/tikv/rawkv_test.go

```diff
@@ -126,6 +132,24 @@ func (s *testRawKVSuite) TestSimple(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testRawKVSuite) TestBatch(c *C) {
+	testNum := 3
+	testKeys := make([][]byte, 0, testNum)
+	testValues := make([][]byte, 0, testNum)
+	for i := 0; i < testNum; i++ {
+		key := fmt.Sprint("key", i)
+		testKeys = append(testKeys, []byte(key))
+		value := fmt.Sprint("value", i)
+		testValues = append(testValues, []byte(value))
+
+		s.mustNotExist(c, []byte(key))
+	}
+	s.mustBatchPut(c, testKeys, testValues)
```

Comment:
Can you test the situation where the batch covers more than one region?

Reply:
yes, the test case should cover more than one regions.

Reply:
Is the test data large enough so that it toggles `size >= limit` in `appendBatches`?

Reply:
Done